boot/zephyr/dts.overlay

```diff
@@ -13,5 +13,13 @@
 	chosen {
 		zephyr,code-partition = &boot_partition;
 	};
+
+#ifdef CONFIG_SOC_INTEL_S1000
```

Comment:
better add  dedicated overlay for this board than put Kconfi postprocesor macro. Zephyr is running away from using kconfig inside of DT.

Reply:
obviously need to modify cmakelists.txtx as well

Reply:
@rgundi ^^

Reply:
@nvlsianpu addressed.

Reply:
I've got error while was building this:
```
-- Loading /home/anpu/codedev/zephyr/boards/xtensa/intel_s1000_crb/intel_s1000_crb.dts as base
-- Overlaying /home/anpu/codedev/zephyr/dts/common/common.dts
-- Overlaying /home/anpu/codedev/mcuboot/boot/zephyr/dts.overlay
-- Overlaying /home/anpu/codedev/mcuboot/boot/zephyr/boards/intel_s1000_crb.overlay
intel_s1000_crb.dts_compiled: ERROR (path_references): /chosen: Reference to non-existent node or label "boot_partition"

ERROR: Input tree has errors, aborting (use -f to force output)
```

Reply:
Not sure why you are getting that error. I tested building for 3 different boards viz. intel_s1000_crb, frdm_k64f and nrf52840_pca10056. It builds fine on all three.
Did you try rebasing?

Reply:
oh.. i get it. You are getting that error because you don't have the corresponding Zephyr changes. There's a PR under review in Zephyr (https://github.com/zephyrproject-rtos/zephyr/pull/11850). Please pull that into your zephyr tree and then compile. It should pass.